Source/PLCrashLogWriter.m

```diff
@@ -49,10 +49,6 @@
 #import "PLCrashSysctl.h"
 #import "PLCrashProcessInfo.h"
 
-#if TARGET_OS_IPHONE
-#import <UIKit/UIKit.h> // For UIDevice
```

Comment:
Should we remove it from project settings somewhere?

Reply:
Fixed by https://github.com/microsoft/plcrashreporter/pull/38